src/components/CustomerDetails/InputPassportSection.tsx

```diff
@@ -51,11 +51,13 @@ export const InputPassportSection: FunctionComponent<InputPassportSection> = ({
     setSelectedCountry(item);
   };
 
+  const trimmedPassportNum: string = (passportNum || "").trim();
```

Comment:
whats the reason for `(passportNum || "")?

Reply:
`passportNum` might be undefined and `passportNum!` leads to test cases failing